### ISSUE TITLE
About    centroids = torch.zeros(3, 304, 64, 64).cuda()  # in line 21 train.py

### ISSUE THREAD

**Issue by guomaotianjie** [EmmaW8/Dofe#2]
Can you tell me how computer 3 domain mean ? I try computer Fundus Data Domain1-3 mean , but it is diffrent with you, thanks.

**Comment by emma-sjwang** [comment]
You can use your own one. It is just utilized for normalization.
